src/oge/data_pipeline.py

```diff
@@ -678,6 +692,15 @@ def main(args):
             ba_fuel_data, year, path_prefix, args.skip_outputs, include_hourly=False
         )
 
+        # export plant static attributes to csv
+        output_data.output_intermediate_data(
+            plant_attributes.assign(shaped_plant_id=np.NaN),
```

Comment:
This should be `pd.NA` since shaped_plant_id should have an Int dtype

Reply:
Done